docs/AboutUs.md

```diff
@@ -19,15 +19,14 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 * Responsibilities: Responsible for the quality of various project documents. 
 In charge of defining, assigning and tracking project tasks.
 
-### Jane Doe
+### Lyndon Lim Liang Hng
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/lyndonlimlianghng.png" width="200px">
```

Comment:
Name of the image should be your github username in lowercase.

Reply:
Thank you. I have fixed it